source/user-manual/reference/ossec-conf/wodle-s3.rst

```diff
@@ -364,7 +364,7 @@ Run evaluation immediately when service is started.
 interval
 ^^^^^^^^
 
-Frequency for reading from the S3 bucket.
+Time the module will wait for before being executed again.
```

Comment:
```suggestion
The amount of time the module will wait for before attempting to run again.
```

Reply:
Added in [6416049](https://github.com/wazuh/wazuh-documentation/pull/4857/commits/64160492ee3a5db0885a2287616659738859d5aa)